source/transactions/tests/write-concern.yml

```diff
@@ -367,8 +357,7 @@ tests:
           session: session0
           filter:
             _id: 0
-        result:
-          deletedCount: 1
+        # avoid assertion since result fields are inaccessible for unacknowledged write concern
```

Comment:
This command is part of a transaction so the driver ignores the w:0 write concern. A result *is* returned.